checkbox-provider-kivu/units/jobs.pxu

```diff
@@ -199,19 +198,13 @@ command:
   echo "{\"gpu_usage_hw_enabled\": $(gpu-load.py --timeout=${PLAY_TIMEOUT} --gpu={{ driver }})}" | tdb.py insert &
   # Play fullscreen looping video in Chromium
   echo XDG_SESSION_TYPE "${XDG_SESSION_TYPE}"
-  sudo --preserve-env -u "${NORMAL_USER}" timeout ${PLAY_TIMEOUT} \
-    chromium \
-    --start-fullscreen \
-    --enable-logging=stderr \
-    /home/"${NORMAL_USER}"/checkbox-test-data/h264-video.html &> \
-    "${PLAINBOX_SESSION_SHARE}"/chromium_h264_decoding_vaapi_enabled.log
+  sudo --preserve-env -u "${NORMAL_USER}" timeout ${PLAY_TIMEOUT} bash -c 'chromium --start-fullscreen \
```

Comment:
This formatting is IMHO worse than what it was before.
If you wanted to have the "internal" command stand out, you should've outlined it in a envvar and use it in the outer invocation.